content/en/synthetics/mobile_app_testing/_index.md

```diff
@@ -26,32 +26,141 @@ cascade:
+
+* Retry `X` times after `Y` ms in case of failure. 
+
+## Scheduling and alerts
+
+By default, mobile app tests are set up for on-demand testing, meaning these tests can run [directly in a CI pipeline](#run-tests-in-ci).
+
+{{< img src="mobile_app_testing/alerting_rules.png" alt="Scheduling and alerting conditions for a mobile test" style="width:90%" >}}
+
+You can customize alert conditions to define how often you want to send an alert and the circumstances under which you want a test to send a notification alert.
+
+* An alert is triggered if any assertion fails for `X` minutes. 
+
+### Configure the test monitor
+
+A notification is sent according to the set of alerting conditions. Use this section to define how and what to message your teams.
+
+1. Enter a **message** for the mobile app test. This field allows standard [Markdown formatting][5] and supports the following [conditional variables][6]:
+
+    | Conditional Variable       | Description                                                         |
+    |----------------------------|---------------------------------------------------------------------|
+    | `{{#is_alert}}`            | Show when the monitor alerts.                                       |
+    | `{{^is_alert}}`            | Show unless the monitor alerts.                                     |
+    | `{{#is_recovery}}`         | Show when the monitor recovers from `alert`.                          |
+    | `{{^is_recovery}}`         | Show unless the monitor recovers from `alert`.                        |
+    | `{{#is_renotify}}`         | Show when the monitor renotifies.                                   |
+    | `{{^is_renotify}}`         | Show unless the monitor renotifies.                                 |
+    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).                  |
+    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).                |
+
+    Notification messages include the **message** defined in this section and information about the failing locations.
+
+2. Choose team members and services to notify.
+3. Specify a renotification frequency. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
+4. Click **Save & Edit Recording** to save your test configuration and record your mobile app test steps.
+
+For more information, see [Using Synthetic Test Monitors][7].
+
+## Run tests in CI
+
+You can run mobile app tests in a CI pipeline by defining the `mobileApplicationVersionFilePath` option in a [test `synthetics.json` file][13] and a [global configuration `synthetics-ci.config` file][14] as needed. Global configuration file options take precedence over test configuration file options.
+
+In this example, the test `aaa-aaa-aaa` runs with the override application version found in `application/path`.
+
+```json
```

Comment:
Would a codeblock with a filename on it be preferred here?

Reply:
That I'm not sure, this content was there before so I just copied it over, it seems the `json` block works ok though.